recipes/avir/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from conans.errors import ConanInvalidConfiguration
+from conans import ConanFile, CMake, tools
```

Comment:
```suggestion
from conans import ConanFile, tools
```

Reply:
Thank you!
I will fix it at avir future release.